model.py

```diff
@@ -1885,8 +1885,15 @@ def author_contributor_tiers(cls):
 
     @classmethod
     def lookup(cls, _db, name=None, viaf=None, lc=None, aliases=None,
-               extra=None):
-        """Find or create a record for the given Contributor."""
+               extra=None, create_new=True):
```

Comment:
What's a scenario where we would pass in create_new=False? Given that we have heard about this contributor from somewhere, I think we would always want a database record to be created.